app/gamelib/src/game_state/changes/controls.rs

```diff
@@ -1,18 +1,50 @@
+use alloc::vec::Vec;
+use alloc::vec;
 /// Represents the state of one or multiple Wii controllers w.r.t. the game
 ///
 /// Its fields are public so they can be filled in from outside the library
 /// (before `GameState::update` is called)
-#[derive(Debug, Default)]
+
 pub struct Controls {
-    pub home_button_down: bool,
-    pub one_button_down: bool,
+    pub wii_mote_control: Vec<WiiMoteControl>
```

Comment:
Neat! We're always going to have 4, so a simple fixed-size array could do too right? Oh, and maybe name it `wii_mote_controllers` (plural indicating an array).